utils/resizingConstraints.js

```diff
@@ -0,0 +1,33 @@
+const containsAllItems = (needles, haystack) => needles.every(needle => haystack.includes(needle));
+
+const resizingConstraintValues = {
+  top: 31,
+  right: 62,
+  bottom: 55,
+  left: 59,
+  width: 61,
+  height: 47,
+  none: 63,
+};
+
+const calculateResizingConstraintValues = (first, ...rest) => {
```

Comment:
It took me a bit of time to understand what this code is doing, could you add a comment that explains how calculating the resize constraint works. Also, the method name and arguments are a bit confusing to me. Why is it using `first, ...rest` and then in the body of the function it turns it back into a single array of args? What about using a single argument that is an array? The method name having "Values" at the end to me makes it seem it will return multiple resizing constraints. Do we need "Values" at all the function name or map, could they be 'resizingConstrains' and 'calculateResizingConstraint'? What do you think?

Reply:
You are right, I took the code from the blog post. I refactored, let me know what do you think.